src/Controller/EnMarche/MunicipalChiefController.php

```diff
@@ -17,12 +23,88 @@ class MunicipalChiefController extends Controller
     use CanaryControllerTrait;
 
     /**
-     * @Route(name="app_municipalchief_index")
+     * @Route(
+     *     path="/municipale/candidature-colistiers",
+     *     name="app_municipal_chief_municipal_running_mate_request",
```

Comment:
No change required here, just a tip our lord and savior @ottaviano taught me:

You can decalre a `name` on the `Route` annotation of the Controller **class**, then it will be prefixed to all the action names.

So you could do :
```
//...

/**
 * @Route(name="app_municipal_chief_")
 */
class MunicipalChiefController
{
    /**
     *  @Route(name="municipal_running_mate_request")
     */
    public function municipalRunningMateRequest(): Response
    {
          //...
    }

    /**
     *  @Route(name="municipal_volunteer_request")
     */
    public function municipalVolunteerRequest(): Response
    {
          //...
    }
}
```

Reply:
And so i'll do :+1:.